.github/workflows/README.md

```diff
@@ -214,6 +215,102 @@ jobs:
+
+```cfg
+[bumpversion]
+current_version = 1.0.1-SNAPSHOT
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(-(?P<suffix>\w+))?
+serialize =
+	{major}.{minor}.{patch}-{suffix}
+	{major}.{minor}.{patch}
+
+[bumpversion:file:path/to/Chart.yaml]
+search =
+	version: {current_version}
+	appVersion: {current_version}
+replace =
+	version: {new_version}
+	appVersion: {new_version}
+```
+
+### Dependencies
+
+This workflow is built from multiple composite actions listed below:
+
+- [bump-version](https://github.com/bakdata/ci-templates/tree/main/actions/bump-version)
+- [commit-and-push](https://github.com/bakdata/ci-templates/tree/main/actions/commit-and-push)
+
+### Input Parameters
+
+| Name                    | Required | Default Value |  Type  | Description                                                                                                    |
+| ----------------------- | :------: | :-----------: | :----: | -------------------------------------------------------------------------------------------------------------- |
+| version-configs-dir     |    ✅    |               | string | The Path to the directory containing the file where the versioning is defined and the `.bumpversion.cfg` file. |
+| release-type            |    ✅    |       -       | string | Scope of the release (major, minor or patch).                                                                  |
+| next-dev-release-type   |    ✅    |       -       | string | Scope of the next release (minor or patch) for developers.                                                     |
+| next-dev-release-suffix |    ❌    |  "SNAPSHOT"   | string | The suffix to add for the developer version.                                                                   |
+
+### Secret Parameters
+
+These secrets define the GitHub user that pushes the changes to the repository. Create a repository secret for the GitHub username (`GH_USERNAME`), the GitHub email (`GH_EMAIL`), and a personal access token (`GH_TOKEN`) of the user.
+You can use the no-reply GitHub email for the email: `[username]@users.noreply.github.com`.
+
+| Name            | Required | Description                                    |
+| --------------- | :------: | ---------------------------------------------- |
+| github-username |    ✅    | The GitHub username for committing the changes |
+| github-email    |    ✅    | The GitHub email for committing the changes    |
+| github-token    |    ✅    | The GitHub token for committing the changes    |
+
+### Calling the workflow
+
+```yaml
+name: Release multiple Helm Charts
+on:
+  workflow_dispatch:
+    inputs:
+      release-type:
+        description: "Scope of the release (major, minor or patch)."
+        required: true
+        type: string
+      next-dev-release-type:
+        description: "Scope of the next release (minor or patch) for developers"
+        required: true
+        type: string
```

Comment:
Please update release-type and next-dev-release-type to use type `choice` instead of `string`

Reply:
isnt it better to let it this way here and let the end user add the  `choice`. Because if we set it to. choice here and for some reason the user just want to always do. `patch` (for instance). Then it might be overwhelming to always have to chose.

Reply:
and i've also realized that `type: choice` cannot be used in `workflow_call` workflows but only in `workflow_dispatch`

Reply:
Ok, then it is fine for me as it is just an example here